ee/clickhouse/queries/test/test_event_query.py

```diff
@@ -216,6 +217,23 @@ def test_account_filters(self):
 
         self._run_query(filter)
 
+    def test_actions(self):
+        person1 = Person.objects.create(team_id=self.team.pk, distinct_ids=["person_1"], properties={"name": "John"})
+        person2 = Person.objects.create(team_id=self.team.pk, distinct_ids=["person_2"], properties={"name": "Jane"})
+
+        _create_event(event="event_name", team=self.team, distinct_id="person_1")
+        _create_event(event="event_name", team=self.team, distinct_id="person_2")
+        _create_event(event="event_name", team=self.team, distinct_id="person_2")
+
+        action = Action.objects.create(team=self.team, name="action1")
+        ActionStep.objects.create(
+            event="event_name", action=action, properties=[{"key": "name", "type": "person", "value": "John"}],
+        )
+
+        filter = Filter(data={"actions": [{"id": action.id, "type": "actions", "order": 0},]})
+
```

Comment:
Is it enough just to test that it runs without error or is there anything to be asserted about the response?

Reply:
Yeah tests in this class are just running it. This crashed w/o app code change.